README.md

```diff
@@ -164,21 +164,25 @@ For edit access, get in touch on
 [Maintainers](https://github.com/open-telemetry/community/blob/main/community-membership.md#maintainer)
 ([@open-telemetry/demo-maintainers](https://github.com/orgs/open-telemetry/teams/demo-maintainers)):
 
-- [Austin Parker](https://github.com/austinlparker), Lightstep
-- [Carter Socha](https://github.com/cartersocha), Microsoft
-- [Morgan McLean](https://github.com/mtwo), Splunk
+- [Carter Socha](https://github.com/cartersocha), Lightstep
+- [Juliano Costa](https://github.com/julianocosta89), Dynatrace
+- [Michael Maxwell](https://github.com/mic-max), Microsoft
 - [Pierre Tessier](https://github.com/puckpuck), Honeycomb
 
 [Approvers](https://github.com/open-telemetry/community/blob/main/community-membership.md#approver)
 ([@open-telemetry/demo-approvers](https://github.com/orgs/open-telemetry/teams/demo-approvers)):
 
-- [Juliano Costa](https://github.com/julianocosta89), Dynatrace
-- [Michael Maxwell](https://github.com/mic-max), Microsoft
+- [Austin Parker](https://github.com/austinlparker), Lightstep
 - [Mikko Viitanen](https://github.com/mviitane), Dynatrace
 - [Penghan Wang](https://github.com/wph95), AppDynamics
 - [Reiley Yang](https://github.com/reyang), Microsoft
 - [Ziqi Zhao](https://github.com/fatsheep9146), Alibaba
 
+[Emeritus
+Maintainer/Approver/Triager](https://github.com/open-telemetry/community/blob/main/community-membership.md#emeritus-maintainerapprovertriager):
+
+- [Morgan McLean](https://github.com/mtwo), Splunk
```

Comment:
@mtwo FYI.

Reply:
All good, I have a standing conflict with the weekly sync, so I haven't attended in a long time!